crates/weaver_semconv/data/event.yaml

```diff
@@ -0,0 +1,181 @@
+groups:
+  - id: device.app.lifecycle
+    stability: experimental
+    type: event
+    name: device.app.lifecycle
+    brief: >
+      This event represents an occurrence of a lifecycle transition on Android or iOS platform.
+    note: >
+      This event identifies the fields that are common to all lifecycle events for android and iOS using
+      the `android.state` and `ios.state` fields. The `android.state` and `ios.state` attributes are
+      mutually exclusive.
+    body:
+      id: device_lifecycle
+      type: map
+      requirement_level: optional
```

Comment:
added `requirement_level` and it is not required at each level (based on the same logic as all others)

Reply:
Do you have a default in the model? (i.e. your requirement level can be set above you in the hierarchy and then SHOULD be excluded below?)  That would make sense to me but we can add in more checks over time.

Reply:
Currently, the AnyValue is required to be present (as per the same behavior from Attributes). But we can make it optional and apply the same logic.